Exercise/src/my_string.cpp

```diff
@@ -0,0 +1,67 @@
+#include "my_string.h"
+
+Exercise_1::CMyString::CMyString(const char* charsSequence) 
+{
+	size_t sCurrentIndex = 0;
+	char cCurrentChar = charsSequence[sCurrentIndex];
+	while(cCurrentChar != '\0')
+	{
+		sCurrentIndex++;
+		cCurrentChar = charsSequence[sCurrentIndex];
+	}
+	m_sSize = sCurrentIndex + 1;
+	m_cData = new char[m_sSize];
```

Comment:
Can new throw an exception here?

Reply:
How should resolve such thing?
I need to allocate array here

Reply:
Fixed